schema/dynamodb/table.json

```diff
@@ -12,7 +12,7 @@
   ],
   "GlobalSecondaryIndexes": [
     {
-      "IndexName": "ClientIDDataTypeMtimeIndex",
+      "IndexName": "ClientIDDataTypeMtimeIndexV2",
```

Comment:
a couple db changes are needed for this to work:

- the creation of this new index (and subsequent deletion of the old index)
- enabling ttl for the `ExpirationTime` attribute